rfcs/0062-content-addressed-paths.md

```diff
@@ -108,118 +156,139 @@ create table if not exists Realisation (
 )
 ```
 
-### Building a non-ca derivation
-
-#### Resolved derivations
+### Resolved derivations
 
 As it is already internally the case in Nix, we define a **basic derivation** as a derivation that doesn't depend on any derivation output (except its own). Said otherwise, a basic derivation is a derivation whose only inputs are either
 
 - Placeholders for its own outputs (from the `placeholder` builtin)
 - Existing store paths
 
-For a derivation `drv` whose input derivations have all been realised, we define its **associated resolved derivation** `resolved(drv)` as `drv` in which we replace every input derivation `inDrv` of `drv` by `Realisation(inDrv).path`, and update the output hash accordingly.
+For a derivation `drv` whose input derivations have all been realised, we define its **associated resolved derivation** `resolved(drv)` as `drv` in which
+we replace every input derivation `inDrv` of `drv` by `Realisation(inDrv).path`.
```

Comment:
I suppose the newline shouldn't be inserted.

Reply:
I’ve never been able to choose beween a strict 80 chars wrapping and wrapping at the end of each sentence when writing markdown code. That one is a weird mix that results in following neither of these options 😑